infra/presubmit.py

```diff
@@ -197,6 +201,7 @@ def do_checks(changed_files):
     '.sh',
     '.yaml',
 ]
+_CHECK_LICENSE_EXCLUSIONS = ['project.yaml']
```

Comment:
should we delete line 202 to skip license checks for all .yaml files?

Reply:
Upto you, i am fine with either ? Which other yaml files you can think of ? Ideally all of them should :(

Reply:
I can't think of any .yaml files in OSS-Fuzz repo that need license check

Reply:
Ok feel free to remove that ?